server/models/Notifications.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require("mongoose");
+const { ObjectId } = mongoose.Schema.Types;
+
+const NotificationSchema = new mongoose.Schema({
+  userId: {
+    type: ObjectId,
+    required: true,
+    ref: 'user',
+  },
+  type: {
+    type: String,
+    enum: [ 'newRequest', 'requestStatus', 'payment' ],
+    required: true,
+  },
+  date: {
```

Comment:
Optional: Mongoose schemas have a timestamps option that tells Mongoose to automatically manage `createdAt` and `updatedAt` properties on your documents

Ref: https://masteringjs.io/tutorials/mongoose/timestamps

Reply:
yes, I think this a better way to manage time